### PR TITLE
(DOCSP-43346) Auditing and Logging

### PR DESCRIPTION
**Pull request by JuliaMongo**
[DOCSP-43346](https://jira.mongodb.org/browse/DOCSP-43346)

@lantoli , @fmenezes , I reused the example from this [auditing.md file](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/c34e0a96f52df398dbddfcb29c84f264e311f222/docs/resources/auditing.md) from the Terraform repo.

Staging link:

[Auditing and Logging](https://deploy-preview-33--docs-atlas-architecture.netlify.app/auditing-logging/)

### CODE REVIEW

File: source/auditing-logging.txt [hunk 1]
```diff
@@ -12,17 +12,77 @@ Auditing and Logging
...
-Content here
+Auditing
+~~~~~~~~
+
+Database auditing lets administrators track system activity for 
+deployments with multiple users. |service| administrators can select 
+the actions, database users, |service| roles, and LDAP groups that they 
+want to audit. You can configure manual :manual:`auditing </core/auditing>` 
+in |service| of most of the documented :manual:`system event actions 
+<audit-action-details-results>`.
```
Comment (lmkerbey-mdb) [inline source/auditing-logging.txt:28]:
```suggestion
want to audit. You can configure manual :manual:`auditing </core/auditing>` 
of most of the documented :manual:`system event actions 
<audit-action-details-results>` in |service|.
```
This reads more naturally to me.

File: source/auditing-logging.txt [hunk 2]
```diff
@@ -12,17 +12,77 @@ Auditing and Logging
...
+
+To learn about best practices for auditing the actions of temporary
+database users, see :atlas:`Audit Temporary Database Users <audit-temp-db-users>`.
+
+By default, |service| performs database authentication auditing in M10+ clusters
+to record authentication events, which include:
+
+- database users
+- source IP addresses
+- timestamps for successful and failed attempts.
```
Comment (lmkerbey-mdb) [inline source/auditing-logging.txt:40]: Database users and IP addresses are not, themselves, authentication events, right? There are authentication events which pertain to them, though. Are the recorded events database user login attempts and connection attempts from IP addresses? I think we should either stick with the characterization 'events' and then find a way to rewrite these items to be events, or we rework the wording above to say something like 'record authentication events, including those pertaining to:'

File: source/auditing-logging.txt [hunk 3]
```diff
@@ -12,17 +12,77 @@ Auditing and Logging
...
+
+In addition, you can configure a JSON-formatted audit filter
+to customize MongoDB auditing.
+For a full list of events you can configure for auditing, and for a list
+of examples, see :manual:`MongoDB auditing </core/auditing>`.
```
Comment (lmkerbey-mdb) [inline source/auditing-logging.txt:45]: RE: the above, consulting this list might be a good way to get an idea of how we can rewrite the list to describe events.

**Review by lmkerbey-mdb — CHANGES_REQUESTED** [review]

**Review by lantoli — APPROVED** [review]

**Review by fmenezes — APPROVED** [review]

File: source/auditing-logging.txt [hunk 4]
```diff
@@ -12,40 +12,117 @@ Auditing and Logging
...
       :tabid: Terraform
+ 
+      The following example demonstrates how to enable auditing for
+      your deployment. Before you can create resources with Terraform,
+      you must:
+
+      - :ref:`Create your paying organization <configure-paying-org>`
+        and :ref:`create an API key <atlas-admin-api-access>` for the
+        paying organization. Store your API key as environment variables
+        by running the following command in the terminal:
```
Comment (lmkerbey-mdb) [inline source/auditing-logging.txt:116]:
```suggestion
        paying organization. Store your public and private keys as environment 
        variables by running the following command in the terminal: 
```

**Review by lmkerbey-mdb — CHANGES_REQUESTED** [review]

**Review by lmkerbey-mdb — APPROVED** [review]
LGTM

**Pull request merged**